### ISSUE TITLE
Download and configure the training data by docker issue

### ISSUE THREAD

**Issue by yishinhung** [google-coral/edgetpu#630]
Hi Coral team:
     I use the following link to training edgetpu model by docker. But, i encounter the following "110 Illegal instruction" issue.
Did you have any idea about this? Thank a lot!

- System enviroment:
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.3 LTS
Release:        20.04
Codename:       focal

- Reference link: https://coral.ai/docs/edgetpu/retrain-detection/#download-and-configure-the-training-data
# Run this from within the Docker container (at tensorflow/models/research/):
./prepare_checkpoint_and_dataset.sh --network_type mobilenet_v1_ssd --train_whole_model false

PREPARING dataset using first two classes of Oxford-IIIT Pet dataset...
+ cp /tensorflow/models/research/learn_pet/pet/annotations/list.txt /tensorflow/models/research/learn_pet/pet/annotations/list_petsdataset.txt
+ cp /tensorflow/models/research/learn_pet/pet/annotations/trainval.txt /tensorflow/models/research/learn_pet/pet/annotations/trainval_petsdataset.txt
+ cp /tensorflow/models/research/learn_pet/pet/annotations/test.txt /tensorflow/models/research/learn_pet/pet/annotations/test_petsdataset.txt
+ grep Abyssinian /tensorflow/models/research/learn_pet/pet/annotations/list_petsdataset.txt
+ grep american_bulldog /tensorflow/models/research/learn_pet/pet/annotations/list_petsdataset.txt
+ grep Abyssinian /tensorflow/models/research/learn_pet/pet/annotations/trainval_petsdataset.txt
+ grep american_bulldog /tensorflow/models/research/learn_pet/pet/annotations/trainval_petsdataset.txt
+ grep Abyssinian /tensorflow/models/research/learn_pet/pet/annotations/test_petsdataset.txt
+ grep american_bulldog /tensorflow/models/research/learn_pet/pet/annotations/test_petsdataset.txt
+ echo 'PREPARING label map...'
PREPARING label map...
+ cd /tensorflow/models/research
+ cp object_detection/data/pet_label_map.pbtxt /tensorflow/models/research/learn_pet/pet
+ echo 'CONVERTING dataset to TF Record...'
CONVERTING dataset to TF Record...
+ python object_detection/dataset_tools/create_pet_tf_record.py --label_map_path=/tensorflow/models/research/learn_pet/pet/pet_label_map.pbtxt --data_dir=/tensorflow/models/research/learn_pet/pet --output_dir=/tensorflow/models/research/learn_pet/pet
**./prepare_checkpoint_and_dataset.sh: line 103:   110 Illegal instruction     (core dumped) python object_detection/dataset_tools/create_pet_tf_record.py --label_map_path="${DATASET_DIR}/pet_label_map.pbtxt" --data_dir="${DATASET_DIR}" --output_dir="${DATASET_DIR}"**

**Comment by hjonnala** [comment]
> 110 Illegal instruction     (core dumped)

I am not able to reproduce this issue. It might be that [TensorFlow](https://tech.amikelive.com/node-887/how-to-resolve-error-illegal-instruction-core-dumped-when-running-import-tensorflow-in-a-python-program/) has crashed even before it does anything.

> ./prepare_checkpoint_and_dataset.sh --network_type mobilenet_v1_ssd --train_whole_model false

Here are the full successful logs for the above command: https://github.com/hjonnala/snippets/blob/main/docker/download_training_data_log.txt#L124

**Comment by yishinhung** [comment]
Hi @hjonnala 
     I found it is related about our cpu don't support the avx by cat /proc/cpuinfo.
I change the cpu supported avx and then don't happen the crash.
Thank your help!
